src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -3475,21 +3493,22 @@ class C
             End Function
         End Class
 
-        <MemberData(NameOf(AllCompletionImplementations))>
+        <InlineData(CompletionImplementation.Legacy)>
+        <InlineData(CompletionImplementation.Modern, Skip:="https://github.com/dotnet/roslyn/issues/29110")>
         <WpfTheory, Trait(Traits.Feature, Traits.Features.Completion)>
         Public Async Function Filters_EmptyList1(completionImplementation As CompletionImplementation) As Task
             Using state = TestStateFactory.CreateCSharpTestState(completionImplementation,
                 <Document><![CDATA[
-using System.IO;
-using System.Threading.Tasks;
-class C
-{
-    async Task Moo()
-    {
-        var x = asd$$
-    }
-}
-            ]]></Document>)
+        using System.IO;
+        using System.Threading.Tasks;
+        class C
+        {
+            async Task Moo()
+            {
+                var x = asd$$
+            }
+        }
+                    ]]></Document>)
```

Comment:
why were these tests edited?

Reply:
We have an open issue not yet resolved: [https://github.com/dotnet/roslyn/issues/29110](https://github.com/dotnet/roslyn/issues/29110)

---
In reply to: [222523119](https://github.com/dotnet/roslyn/pull/29016#discussion_r222523119) [](ancestors = 222523119)

Reply:
❗️ Changes to test should be reverted